RLTSquare/Ccq/Console/Command/HelloWorld.php

```diff
@@ -0,0 +1,127 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Exception;
+use Magento\Framework\MessageQueue\PublisherInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use RLTSquare\Ccq\Api\Data\CcqInterface;
+
+/**
+ * @author Afzel Arshad
+ * Cli command for message queue
+ */
+class HelloWorld extends Command
+{
+    /**
+     *  Topic name
+     */
+    const TOPIC_NAME = 'rltsquare.hello.world';
```

Comment:
Code refactoring - might have been better if this constant sit in consumer class instead of here; then you just call it as Consumer::TOPIC_NAME.